main/src/mill/modules/CoursierSupport.scala

```diff
@@ -306,4 +307,32 @@ object CoursierSupport {
     }
   }
 
+  def repoFromString(str: String, origin: String): Option[Seq[Repository]] = {
+    val spaceSep = "\\s+".r
+
+    val l =
```

Comment:
Can we have a better name than `l` here? Maybe `repositoryStringsToParse` or something? AFAICT it's basically the half-parsed repository names that we split before passing to Coursier.

Also, what's the reason we split them differently depending on `spaceSep.findFirstIn(str).isEmpty`? I assume there's a good reason, but probably should go in a comment somewhere

Reply:
This code is copy-paste as-is from Ammonite and I just wanted the import working. Feel free to change it as you want.

Reply:
Done